specs/sb/api.md

```diff
@@ -364,6 +435,29 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         "bullets": [{
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instances": {
```

Comment:
As mentioned during the call these examples would actually validate successfully against everything that is not of type `object`, e.g. `42` or `foo`. 

This is because we they don't specify a `type` constraint, i.e. they allow any type. The `properties` and `required` constraints are only enforced for types for which they apply, i.e. the `object` type. They don’t apply for other types. In other words, the says I’m okay with any type, but if it is an `object` it has to have a property with name `fake_required_property` and type `string`.

You can try that yourself using one of the openly available JSON-schema validators, e.g. http://www.jsonschemavalidator.net and using the following two schemas ...

```
{"properties": { "a": {} }, "required": [ "a" ]}
```
and
```
{"type": "string", "properties": { "a": {} }, "required": [ "a" ]}
```

Number `42` successfully validates against the first but not the second schema.

I agree that this behavior is kinda weird. But it works as designed. See https://github.com/json-schema/json-schema/issues/172.